dashboard/src/features/tutorial/components/Success/Success.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styles from './Success.scss'
+import { Link } from 'react-router'
+
+class Success extends React.Component {
+
+  render() {
+    const userInput = this.props.userInput
+    const nextButton = <div className={styles.next}>
+      <Link to={this.props.route}>
+        <button key='showNext' className='btn btn-primary' onClick={this.props.handleNext}>
+          {this.props.button}
+        </button>
+      </Link>
+    </div>
+
+    return (
+      <div>
+        <div className={styles.container}>
+          <div className={styles.header}>
+            {this.props.title}
+            {this.props.dismiss &&
+              <div className={styles.skip}>
+                <a onClick={this.props.handleDismiss}>{this.props.dismiss}</a>
+              </div>
+            }
+          </div>
+          <div className={styles.content}>
+            <span className='glyphicon glyphicon-ok-sign'></span>
+            <div className={styles.text}>
+              {this.props.content.map(function (x, i){
+                let str = x['line']
+                if (x['type']) {
+                  if (x['type'] == 'account'){
```

Comment:
You could flip this logic around a little bit. Something like:

```js
let replacement = userInput[x['type']]
if (x['index']) {
  replacement = replacement[x['index']]
}
str = str.replace(replacement['alias'])
```

would basically only go down the `index` path if it's present (and works if something other than accounts uses multiple values in the future).